tests/ftqc/test_parametric_mid_measure.py

```diff
@@ -480,6 +480,100 @@ def test_measure_z_dispatches_to_measure(self, reset, postselect):
         assert mp.postselect == postselect
         assert isinstance(mp, MidMeasureMP)
 
+    # pylint: disable=too-many-positional-arguments, too-many-arguments
+    @pytest.mark.jax
+    @pytest.mark.usefixtures("enable_disable_plxpr")
+    @pytest.mark.parametrize(
+        "meas_func, angle, plane", [(measure_x, 0.0, "XY"), (measure_y, 1.5707, "XY")]
```

Comment:
Can we add some non-standard numerics too?
As in, no use of 0,1, pi, pi/2, etc

Reply:
This one only tests `measure_x` and `measure_y`, so these are set values - but `measure_arbitrary_basis` is tested with 1.23 and -0.34 below. 

The tests are almost identical but the differing call signature on `measure_arbitrary_basis` made the test parametrization awkward if they were in the same test function.

Reply:
Sounds good. I'll resolve this.